westpa_wexplore/wexplore.py

```diff
@@ -2,7 +2,8 @@
 import networkx as nx
 import pandas as pd
 import heapq
-import cPickle as pickle
+#KFW import cPickle as pickle
```

Comment:
This could probably be better handled by:

```python
try:
    import cPickle as pickle
except ImportError:
    import pickle
```

I guess this depends on whether we want to maintain python 2/3 compatibility or if we're dropping python2 completely.